tcn_hpl/models/ptg_module.py

```diff
@@ -294,7 +290,7 @@ def on_validation_epoch_end(self) -> None:
 
         # Save results
         dset = kwcoco.CocoDataset()
-        dset.fpath = f"{self.output_dir}/val_activity_preds_epoch{self.current_epoch}.mscoco.json"
+        dset.fpath = f"{self.hparams.output_dir}/val_activity_preds_epoch{self.current_epoch}.mscoco.json"
```

Comment:
I'm not sure this will resolve when `output_dir` is set to `None`?

Reply:
Would this ever get called with output_dir set to None? It's is always set on the training side

Reply:
I don't imagine I'll be setting this during inference unless you have specific guidance on what I would be outputting when inferencing.

Reply:
Yeah I think it's fine, you shouldn't be hitting any codepaths that use `output_dir` during inference